### ISSUE TITLE
official binary package with _GLIBCXX_USE_CXX11_ABI=1

### ISSUE THREAD

**Issue by guoyejun** [pytorch/pytorch#72907]
### 🐛 Describe the bug

there are lots of discussions about _GLIBCXX_USE_CXX11_ABI, such as https://github.com/pytorch/pytorch/issues/51039.

Looks that the PyTorch binary installed with 'pip install torch==1.10+cpu -f ...' is still built with _GLIBCXX_USE_CXX11_ABI=0 (see detail below). It is very important for us to get the pytorch binary built with _GLIBCXX_USE_CXX11_ABI=1 for many cases, we want to make our project work with pytorch’s official binary package without re-building pytorch from source code locally.

```
$ python -c 'import torch; print(torch._C._GLIBCXX_USE_CXX11_ABI)'
False

$ nm -C ./lib/libc10.so | grep "Error(std::"
00000000000372b0 T c10::Error::Error(std::string, std::string, void const*)
00000000000372b0 T c10::Error::Error(std::string, std::string, void const*)
000000000004f640 T caffe2::detail::_ThrowRuntimeTypeLogicError(std::string const&)
000000000005ba10 r caffe2::detail::_ThrowRuntimeTypeLogicError(std::string const&)::__func__
```

So, the question is: will pytorch binary with _GLIBCXX_USE_CXX11_ABI=1 be provided in near future (for example, next pytorch version 1.12.0/1.13.0)? Thanks.

### Versions

a general issue for all the versions

cc @ezyang @seemethere @malfet

**Comment by zou3519** [comment]
@malfet @seemethere what do you think?

**Comment by rgommers** [comment]
@guoyejun I think this request is just a duplicate of gh-51039? Can we close this?



> So, the question is: will pytorch binary with _GLIBCXX_USE_CXX11_ABI=1 be provided in near future (for example, next pytorch version 1.12.0/1.13.0)?

I'm not responsible for this, but I think the answer is "no" - not until `manylinux2014` is dropped at least. And right now we're still in the "need to drop `manylinux1`" state.

**Comment by guoyejun** [comment]
thanks for the info, it is in progress, I will close this issue, thanks.
